src/patternfly/components/Nav/examples/nav-tertiary-list-example.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> nav nav--attribute='aria-label="Tertiary Example"' aria-labelledby="[id of parent]"}}
+{{#> nav aria-label="Local" aria-labelledby="[id of parent]"}}
```

Comment:
would this aria-label not be included inside a nav--attribute="" for consistency